README.md

```diff
@@ -519,6 +519,7 @@ issue](https://github.com/brigade/overcommit/issues/238) for more details.
 * [LocalPathsInGemfile](lib/overcommit/hook/pre_commit/local_paths_in_gemfile.rb)
 * [Mdl](lib/overcommit/hook/pre_commit/mdl.rb)
 * [`*`MergeConflicts](lib/overcommit/hook/pre_commit/merge_conflicts.rb)
+* [MetadataJsonLint](lib/overcommit/hook/pre_commit/metadata_json_lint.rb)
```

Comment:
Since this tool is specific to the Puppet ecosystem, let's name it `PuppetMetadataJsonLint` to reflect this.